hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -143,6 +143,7 @@ public final class OzoneConsts {
       "flushBeforeCheckpoint";
   public static final String OZONE_DB_CHECKPOINT_INCLUDE_SNAPSHOT_DATA =
       "includeSnapshotData";
+  public static final String OZONE_DB_CHECKPOINT_REQUEST_SST = "usedSst";
```

Comment:
OZONE_DB_CHECKPOINT_REQUEST_SST -> OZONE_DB_CHECKPOINT_EXCLUDE_SST
usedSst -> excludeSst

Reply:
Renamed to OZONE_DB_CHECKPOINT_REQUEST_TO_EXCLUDE_SST = "toExcludeSST"